docs/pages/eas-update/debug.mdx

```diff
@@ -82,6 +82,18 @@ The displayed deployment has the correct channel, but it is not linked to a bran
 
 If our deployment is not displayed, it means our build is not configured properly for EAS Update. To fix this, [configure our channel](/#configure-channel), [configure our runtime version](/#configure-runtime-version) and verify our [general configuration](/eas-update/debug-advanced/#verifying-app-configuration). We'll need to rebuild our app after making these changes.
 
+### Automatic roll back when an update crashes
+
+If everything looks correct on the deployments page, but your app still shows the previous update or the code embedded with the build, your new update's code may be crashing. This can happen when this new update crashes before the root component renders after the app downloads and applies the new update.
```

Comment:
```suggestion
If everything looks correct on the Deployments page, but your app still shows the previous update or the code embedded with the build, your new update's code may be crashing. This can happen when this new update crashes before the root component renders after the app downloads and applies the new update.
```

I made a small error in my previous suggestion. We should capitalize "Deployments" for consistency on this page since this is a reference of a page on Expo dashboard.